### ISSUE TITLE
Ability to open multiple windows

### ISSUE THREAD

**Issue by pervoj** [linuxmint/xed#420]
```
 * Xed version: 2.6.2
 * Distribution: Linux Mint 20 Ulyana - Cinnamon edition
```

Add the ability to open multiple windows, please. xed will be a little better again!

**Comment by aleex5** [comment]
should add an option in the app preferences to choose whether to open in tabs or in separate windows

**Comment by mtwebster** [comment]
There are a few ways you can accomplish this now - if you *always* want xed to use a new window when opening a document, try:

- Right-click in Nemo, Open With, Other application...
- In the custom entry at the bottom of the window, type:
      `xed --new-window`
- Click 'Set as default'

This can be done in other desktop environments also, but the steps will differ.

----
You can also use [Nemo Actions](https://github.com/linuxmint/nemo/wiki/Nemo-Actions) to add 'open in new window' to your context menu.

**Comment by tschirilieb-blip** [comment]
Yes, that would be really great if you could turn it off. It's very annoying and damaging. I just wonder what you're thinking when you prescribe something like this without giving the user the chance to turn it off. It has already cost me a lot of money and working time. That's because you just keep forgetting that there are several tabs when working and then lose the data when you close the window together with the others. That you have opened files twice, which also leads to data loss. It also hinders my work because I often have to see both or all 3 documents at the same time. It is of no use at all. What should I do with an open document that I can't see? If I don't see it but leave it open, I can hide it myself. I think it's great that someone programmed XED, but thinking about knowing what's right for others is always very problematic. Thank you for the great program, please do it so that you can turn it off. In the meantime, I have corrected this in a cumbersome way via the system in which I force the opening of a new window when the launcher is called. But please, it's like putting on your pants with pliers.

**Comment by tschirilieb-blip** [comment]
What if you can't use the normal server of a software, because otherwise documents will be opened in TABS? I first have to create a script that intercepts the calls throughout the system and forces the software to open a new instance every time. Especially with a universal text program that you want to call up quickly from other applications. This is simply a mistake in user ergonomics. That's wrong,
